sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -2333,7 +2333,7 @@ case class Get(
        b
   """,
   since = "2.4.0",
-  group = "map_funcs")
+  group = "collection_funcs")
```

Comment:
https://github.com/apache/spark/blob/2aa06fcf1607bbad9e09649e587493032e739e35/sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/ExpressionDescription.java#L110-L116

From the comments of `ExpressionDescription#group`, it seems we shouldn't directly use `collection_funcs`. Perhaps we should amend this comment. On the other hand, maybe the grouping of other functions needs correction, for instance, should `Concat` be categorized under `array_funcs`?

Reply:
I don't think it means `collection_funcs` is forbidden.

`Concat` works with multiple types, including string, binary and array, so it's not just an array function;

As I mentioned above, it looks like a `collection function` supports collection type (map/array/struct/etc) and some other types.